tests/test_enterprise/api/test_views.py

```diff
@@ -4977,3 +4989,54 @@ def test_put_method_not_allowed(self):
         self.assertEqual(response.status_code, 405)
         response = response.json()
         self.assertEqual(response['detail'], 'Method "PUT" not allowed.')
+
+    def test_successful_link(self):
```

Comment:
Should we also have test case(s) for when an existing EnterpriseCustomerUser with `linked=False` and `active=False` calls `/link-user/` action, asserting those fields on their existing record should be flipped to `True`?

Reply:
Good catch, we check for both now!